### ISSUE TITLE
In high contrast white theme, focus indication provided for Inverted Link does not satisfy the minimum contrast ratio of 7:1 

### ISSUE THREAD

**Issue by Lokanathan-k** [SAP/fundamental-styles#2697]
Observed Behavior:
In high contrast white theme, focus indication provided for Inverted Link does not satisfy the minimum contrast ratio of 7:1 

Expected Behavior:
Ideally, in high contrast white theme, the contrast ratio should be equal to or more than 7:1

Screenshot:
![image](https://user-images.githubusercontent.com/88035260/127949713-1b8480ca-12c5-4941-8e27-c41dda53442a.png)

**Comment by droshev** [comment]
@Lokanathan-k this issue is only for ngx doc site. there the example has a hardcoded color:
```
<div style="background-color:#314a5e;padding:10px">
    <a fd-link [routerLink]="['./']" fragment="inverted" [inverted]="true">Inverted Link</a>
</div>
```
the example has to be updated
